From 1fb258cea6ffd46e05b19d0985798b5397fd4354 Mon Sep 17 00:00:00 2001
From: Yury Martynov <email@linxon.ru>
Date: Fri, 15 Mar 2019 13:35:13 +0300
Subject: [PATCH] Fix exiv2 issues (https://github.com/linuxmint/pix/issues/70)

---
 extensions/exiv2_tools/exiv2-utils.cpp | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/extensions/exiv2_tools/exiv2-utils.cpp b/extensions/exiv2_tools/exiv2-utils.cpp
index 125df2b..d479454 100644
--- a/extensions/exiv2_tools/exiv2-utils.cpp
+++ b/extensions/exiv2_tools/exiv2-utils.cpp
@@ -28,13 +28,18 @@
 #include <exiv2/error.hpp>
 #include <exiv2/image.hpp>
 #include <exiv2/exif.hpp>
+#include <exiv2/exiv2.hpp>
 #include <iostream>
 #include <iomanip>
 #include <string>
 #include <sstream>
 #include <vector>
 #include <iomanip>
+#if EXIV2_TEST_VERSION(0, 27, 0)
+#include <exiv2/xmp_exiv2.hpp>
+#else
 #include <exiv2/xmp.hpp>
+#endif
 #include <pix.h>
 #include "exiv2-utils.h"
 
@@ -947,7 +952,11 @@ dump_exif_data (Exiv2::ExifData &exifData,
 
 	try {
 		if (exifData.empty()) {
+#if EXIV2_TEST_VERSION(0, 27, 0)
+			throw Exiv2::Error(Exiv2::kerErrorMessage, " No Exif data found in the file");
+#else
 			throw Exiv2::Error(1, " No Exif data found in the file");
+#endif
 		}
 		Exiv2::ExifData::const_iterator end = exifData.end();
 		for (Exiv2::ExifData::const_iterator i = exifData.begin(); i != end; ++i) {
