From 6fc48697c5ba90808e38a920b2f08d317ab7befe Mon Sep 17 00:00:00 2001
From: Kasra_mp <kasra_mp@yahoo.com>
Date: Sat, 11 Aug 2018 10:58:13 +0200
Subject: [PATCH] #69 Fix typo

---
 toot/ui/app.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toot/ui/app.py b/toot/ui/app.py
index d9fb784..71f28fa 100644
--- a/toot/ui/app.py
+++ b/toot/ui/app.py
@@ -298,7 +298,7 @@ def get_content(self):
             "  v      - view current toot in browser",
             "  q      - quit application",
             "",
-            "Press q to exist help.",
+            "Press q to exit help.",
             "",
             ("https://github.com/ihabunek/toot", Color.YELLOW),
         ]

From 9ac1f4d7b4d23f9c13a332e9c781c1d0b900f048 Mon Sep 17 00:00:00 2001
From: Mateusz Piotrowski <0mp@FreeBSD.org>
Date: Tue, 21 Aug 2018 00:42:46 +0200
Subject: [PATCH] Fix a typo

---
 README.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.rst b/README.rst
index e3248f8..29daf0e 100644
--- a/README.rst
+++ b/README.rst
@@ -29,7 +29,7 @@ Features
 * Support for media uploads, spoiler text, sensitive content
 * Search by account or hash tag
 * Following, muting and blocking accounts
-* Simple swithcing between authenticated in Mastodon accounts
+* Simple switching between authenticated in Mastodon accounts
 
 Curses UI
 ---------
