diff -ur a/src/rsensor.c b/src/rsensor.c
--- a/src/rsensor.c	2016-12-20 02:39:54.000000000 +0300
+++ b/src/rsensor.c	2019-02-08 20:57:21.732936697 +0300
@@ -133,7 +133,7 @@
 
 	obj = get_json_object(url);
 
-	if (obj && !is_error(obj)) {
+	if (obj && !obj ) {
 		n = json_object_array_length(obj);
 		sensors = malloc((n + 1) * sizeof(struct psensor *));
 
@@ -168,12 +168,12 @@
 
 	obj = get_json_object(get_url(s));
 
-	if (obj && !is_error(obj)) {
+	if (obj && !obj) {
 		json_object *om;
 
 		json_object_object_get_ex(obj, "last_measure", &om);
 
-		if (!is_error(obj)) {
+		if (!obj) {
 			json_object *ov, *ot;
 			struct timeval tv;
 
