diff -ur a/src/hotcorner.c b/src/hotcorner.c
--- a/src/hotcorner.c	2015-06-30 10:58:39.000000000 +0300
+++ b/src/hotcorner.c	2017-08-14 21:07:36.960476973 +0300
@@ -216,7 +216,7 @@
 static HotCorner * hotCorner_new(XfcePanelPlugin *plugin) {
     HotCorner * hotCorner = g_new0(HotCorner, 1);
     hotCorner->plugin = plugin;
-    hotCorner->icon = xfce_panel_image_new_from_source ("xfce4-display");
+    hotCorner->icon = xfce_panel_image_new_from_source ("");
     hotCorner->disableWhenFullScreen = TRUE;
     hotCorner->upperLeftCallback  = NULL;
     hotCorner->lowerLeftCallback  = NULL;
