From 9bd597102f49a89ca82d5ac3dc21aeb6bf7b96dc Mon Sep 17 00:00:00 2001
From: Sly_tom_cat <slytomcat@mail.ru>
Date: Thu, 22 Mar 2018 15:26:44 +0300
Subject: [PATCH] fix for #191

---
 yandex-disk-indicator.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/yandex-disk-indicator.py b/yandex-disk-indicator.py
index 3a11315..19a6758 100644
--- a/yandex-disk-indicator.py
+++ b/yandex-disk-indicator.py
@@ -337,7 +337,7 @@ def process_IN_MODIFY(self, event):
           handler(par)
       self.watchMngr = WatchManager()    # Create watch manager
       # Create PyiNotifier
-      self.iNotifier = ThreadedNotifier(self.watchMngr, EH(), timeout=0.5)
+      self.iNotifier = ThreadedNotifier(self.watchMngr, EH(), timeout=500)
       self.iNotifier.start()
       self.status = False
 
