diff -Nur a/panel-plugin/Makefile.am b/panel-plugin/Makefile.am
--- a/panel-plugin/Makefile.am	2014-04-09 21:07:46.000000000 +0200
+++ b/panel-plugin/Makefile.am	2017-08-21 21:06:50.477247070 +0200
@@ -22,6 +22,7 @@
 libmixer_la_CFLAGS =							\
 	$(PLATFORM_CFLAGS)						\
 	$(GLIB_CFLAGS)							\
+	$(DBUS_GLIB_CFLAGS)						\
 	$(GTK_CFLAGS)							\
 	$(LIBXFCE4UTIL_CFLAGS)						\
 	$(LIBXFCE4UI_CFLAGS)						\
@@ -43,6 +44,7 @@
 
 libmixer_la_LIBADD =							\
 	$(GLIB_LIBS)							\
+	$(DBUS_GLIB_LIBS)						\
 	$(GTK_LIBS)							\
 	$(LIBXFCE4UTIL_LIBS)						\
 	$(LIBXFCE4UI_LIBS)						\
diff -Nur a/panel-plugin/Makefile.in b/panel-plugin/Makefile.in
--- a/panel-plugin/Makefile.in	2014-04-09 21:40:22.000000000 +0200
+++ b/panel-plugin/Makefile.in	2017-08-21 21:07:51.410109697 +0200
@@ -410,6 +410,7 @@
 libmixer_la_CFLAGS = \
 	$(PLATFORM_CFLAGS)						\
 	$(GLIB_CFLAGS)							\
+	$(DBUS_GLIB_CFLAGS)						\
 	$(GTK_CFLAGS)							\
 	$(LIBXFCE4UTIL_CFLAGS)						\
 	$(LIBXFCE4UI_CFLAGS)						\
@@ -431,6 +432,7 @@
 
 libmixer_la_LIBADD = \
 	$(GLIB_LIBS)							\
+	$(DBUS_GLIB_LIBS)						\
 	$(GTK_LIBS)							\
 	$(LIBXFCE4UTIL_LIBS)						\
 	$(LIBXFCE4UI_LIBS)						\
diff -Nur a/xfce4-mixer/Makefile.am b/xfce4-mixer/Makefile.am
--- a/xfce4-mixer/Makefile.am	2014-04-09 21:07:46.000000000 +0200
+++ b/xfce4-mixer/Makefile.am	2017-08-21 21:06:50.477247070 +0200
@@ -28,6 +28,7 @@
 
 xfce4_mixer_CFLAGS = 							\
 	$(GLIB_CFLAGS)							\
+	$(DBUS_GLIB_CFLAGS)						\
 	$(GTK_CFLAGS)							\
 	$(UNIQUE_CFLAGS)						\
 	$(LIBXFCE4UTIL_CFLAGS)						\
@@ -41,6 +42,7 @@
 xfce4_mixer_LDFLAGS = 							\
 	$(top_builddir)/libxfce4mixer/libxfce4mixer.la			\
 	$(GLIB_LIBS)							\
+	$(DBUS_GLIB_LIBS)						\
 	$(GTHREAD_LIBS)							\
 	$(GTK_LIBS)							\
 	$(UNIQUE_LIBS)							\
diff -Nur a/xfce4-mixer/Makefile.in b/xfce4-mixer/Makefile.in
--- a/xfce4-mixer/Makefile.in	2014-04-09 21:40:22.000000000 +0200
+++ b/xfce4-mixer/Makefile.in	2017-08-21 21:08:59.099588016 +0200
@@ -424,6 +424,7 @@
 
 xfce4_mixer_CFLAGS = \
 	$(GLIB_CFLAGS)							\
+	$(DBUS_GLIB_CFLAGS)						\
 	$(GTK_CFLAGS)							\
 	$(UNIQUE_CFLAGS)						\
 	$(LIBXFCE4UTIL_CFLAGS)						\
@@ -437,6 +438,7 @@
 xfce4_mixer_LDFLAGS = \
 	$(top_builddir)/libxfce4mixer/libxfce4mixer.la			\
 	$(GLIB_LIBS)							\
+	$(DBUS_GLIB_LIBS)						\
 	$(GTHREAD_LIBS)							\
 	$(GTK_LIBS)							\
 	$(UNIQUE_LIBS)							\
