--- a/getsploit.py	2018-02-10 14:47:54.722530680 +0300
+++ b/getsploit.py	2018-02-10 14:45:12.073609287 +0300
@@ -28,6 +28,7 @@
 import unicodedata
 import re
 import os
+from pathlib import Path
 
 vulnersURL = {
     'searchAPI' : 'https://vulners.com/api/v3/search/lucene/',
@@ -66,9 +67,13 @@
     bytes_type = str
 
 
-DBPATH, SCRIPTNAME = os.path.split(os.path.abspath(__file__))
+SCRIPTNAME = os.path.split(os.path.abspath(__file__))
+DBPATH = str(Path.home()) + '/.getsploit'
 DBFILE = os.path.join(DBPATH, 'getsploit.db')
 
+if not os.path.exists(DBPATH):
+    os.makedirs(DBPATH, exist_ok=True)
+
 try:
     import sqlite3
     import zipfile
