diff -ur a/SConscript b/SConscript
--- a/SConscript	2011-05-16 08:44:23.000000000 +0400
+++ b/SConscript	2019-01-14 11:54:49.813334684 +0300
@@ -33,6 +33,7 @@
 	env['CXX'] = os.environ['CXX']
 if os.environ.has_key('CXXFLAGS'):
 	env['CXXFLAGS'] += SCons.Util.CLVar(os.environ['CXXFLAGS'])
+	env.Append(CXXFLAGS = '-std=gnu++98')
 if os.environ.has_key('LDFLAGS'):
 	env['LINKFLAGS'] += SCons.Util.CLVar(os.environ['LDFLAGS'])
 
@@ -114,7 +115,6 @@
 env.Alias(target="install", source=[
 	env.InstallProgram(dir=env['DESTDIR'] +'/bin', source=[executable]),
 	env.InstallData(dir=env['DESTDIR'] +'/share/applications', source=['share/applications/gpick.desktop']),
-	env.InstallData(dir=env['DESTDIR'] +'/share/doc/gpick', source=['share/doc/gpick/copyright']),
 	env.InstallData(dir=env['DESTDIR'] +'/share/gpick', source=[env.Glob('share/gpick/*.png'), env.Glob('share/gpick/*.lua'), env.Glob('share/gpick/*.txt')]),
 	env.InstallData(dir=env['DESTDIR'] +'/share/man/man1', source=['share/man/man1/gpick.1']),
 	env.InstallData(dir=env['DESTDIR'] +'/share/icons/hicolor/48x48/apps/', source=[env.Glob('share/icons/hicolor/48x48/apps/*.png')]),
